### ISSUE TITLE
Isn't doing anything 

### ISSUE THREAD

**Issue by benyanke** [mariusae/trickle#18]
I'm in the process of finding bandwidth shaping tools to use on a server-sync project, and I'm having trouble getting this to work.

I ran `sudo trickled -d 250 -u 250` and my running SCP transfer still continued to chuck along at 100+ MB/s. 

Any ideas?

**Comment by YuGiOhJCJ** [comment]
Same problem here with trickle 66551ad94ad3d8af83e1e4422804676ac8762f47 built from source code on Slackware64 14.2:
```
$ trickle -sd 200 git clone git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
Cloning into 'linux'...
remote: Counting objects: 5626857, done.
remote: Compressing objects: 100% (1330/1330), done.
Receiving objects:   0% (11382/5626857), 5.19 MiB | 662.00 KiB/s
```
I set a maximum download rate to 200 KB/s but the full bandwidth is used (662 KiB/s).
Why please?

**Comment by alglez** [comment]
I'm experiencing the same problem in Arch.

**Comment by DracoBlue** [comment]
trickle works only for non-forking processes. if you try with e.g. curl it works :)

**Comment by phil294** [comment]
> trickle works only for non-forking processes. if you try with e.g. curl it works :)

No, unfortunately, this does not help. Tried `wget`, `curl`, and `pacman -Syyu`. The weird thing is, sometimes it is throttled, sometimes not (in my case, at least). Not sure what is going on here.

**Comment by tbertels** [comment]
Try https://github.com/magnific0/wondershaper
For instance
`sudo wondershaper -a [interface_name] -d 250 -u 250`
To clear limits:
`sudo wondershaper -c -a [interface_name]`
